siocli/lib/SqlDump.cpp

```diff
@@ -36,43 +36,111 @@ namespace {
 
 using namespace siodb::iomgr::dbengine;
 
+/**
+ * Database information
+ */
 struct DatabaseInfo {
+    /** Initializes structure DatabaseInfo. */
+    DatabaseInfo() = default;
```

Comment:
noexcept

Reply:
std::string() is not noexcept

Reply:
/home/alexey/siodb_demilivor/siodb/siocli/lib/SqlDump.cpp:390:22: error: use of deleted function ‘siodb::siocli::{anonymous}::DatabaseInfo::DatabaseInfo()’
         DatabaseInfo database;
                      ^~~~~~~~

BinaryValue constructor is not noexcept

Reply:
Please see my last 2 comments

Reply:
@Demilivor Tried to compile on centos7, and getting this error. 
So one more thing required - please change this particular constructor to 
```c++
    DatabaseInfo() noexcept
    {
    }
```
Looks like gcc-8 still have non-noexcept `std::basic_string::basic_string()`, although actually it is.